search/controllers/advanced/tests.py

```diff
@@ -1,7 +1,7 @@
 """Tests for advanced search controller, :mod:`search.controllers.advanced`."""
 
 from http import HTTPStatus
-from unittest import TestCase, mock
+from unittest import TestCase, mock, skip
```

Comment:
It looks like skip is newly imported but I don't see any changes to this file where it would be used.

Is it be removed?